msg/js/en.js

```diff
@@ -45,6 +45,7 @@ Blockly.Msg["DATA_INDEX_RANDOM"] = "random";
 Blockly.Msg["EVENT_WHENFLAGCLICKED"] = "when %1 clicked";
 Blockly.Msg["EVENT_WHENTHISSPRITECLICKED"] = "when this sprite clicked";
 Blockly.Msg["EVENT_WHENSTAGECLICKED"] = "when stage clicked";
+Blockly.Msg["EVENT_WHENTOUCHINGOBJECT"] = "when this sprite touches %1";
```

Comment:
Isn't this file being gitignored now? Should these changes be in this PR?

Reply:
There is a `!` in the modified `.gitignore` for `en.json` and `en.js.`.

Reply:
Oh right! I guess I might have been responding to an offline conversation with @chrisgarrity instead of the actual .gitignore file. Oops. =]

Reply:
The en.{js, json} files have always been checked in, so I just tried to stay consistent.